### DISCUSSION TITLE
Non repeating colors for line graphs

### DISCUSSION THREAD

**Discussion by prachinerik** [plouc/nivo#2123] [Q&A]
Hello I am trying to use Nivo line to draw a line graph with 100-150 lines, I want each of these lines to have a different color. Is it possible in @nivo line? My aplogies if its obvious. I tried using different color schemes but couldn't do it.

**Comment by gillallifam** [comment]
This is hard to achieve!
Some ideas,
Group lines in  colors gradients
Use custom line styles https://nivo.rocks/storybook/?path=/story/line--custom-line-style
For this amount of colors maybe require both ideas and that is not garanted to work.
This content is translated

**Comment by plouc** [comment]
@prachinerik, the default color schemes don't provide that many colors, and I think it's a bit difficult to find this amount of distinct colors, but technically, it is possible to pass an array of colors to the `Line` component via `colors`, the hardest part will be to find those I'm afraid.

**Comment by plouc** [comment]
I'm closing this issue as it's technically possible to pass an array of non repeating colors, but the library doesn't support so many variations by default, which would be very difficult to achieve anyway, please let me know if you have questions about how to pass those colors.
